### PR TITLE
addProfile

### PR DESCRIPTION
**Pull request by ben-dh3**
Added my profile!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1675"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! Thanks for contributing!! 🚀 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Njong392 — APPROVED** [review]
Looks amazing :100: Thanks for contributing. Isn't it a bit odd to add the link to your linktree in your linktree? (this is totally optional).

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

File: public/data/ben-dh3.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Benjamin Duffield-Harding",
+  "type": "personal",
+  "bio": "Crypto, generative-art, and open source enthusiast",
+  "avatar": "https://github.com/ben-dh3.png",
+  "url": "https://linkfree.eddiehub.io/benjaminduffieldharding",
```
Comment (Panquesito7) [inline public/data/ben-dh3.json:6]: AFAIK, this field should be removed. There's no `url` field in this part.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by imlakshay08 — APPROVED** [review]

**Review by maharshi-sinha — APPROVED** [review]

File: public/data/ben-dh3.json (hunk 1 above)
Comment (ben-dh3) [reply public/data/ben-dh3.json]: Thanks for your help! :)

**Review by Panquesito7 — APPROVED** [review]
Looks good.Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Reusable

**Pull request merged**